library/Fission/Internal/MonadDB/Class.hs

```diff
@@ -0,0 +1,19 @@
+{-# LANGUAGE UndecidableInstances #-}
```

Comment:
Unfortunately need to turn off the automatic decidablilty check. It can't mechanically tell that `MonadDB` has fewer members than `MonadIO`, but we know this is the case (because we're writing the `MonadDB`), so it's okay.